_studio/mfx_lib/encode_hw/hevc/linux/base/hevcehw_base_va_lin.cpp

```diff
@@ -191,6 +195,17 @@ void DDI_VA::InitAlloc(const FeatureBlocks& /*blocks*/, TPushIA Push)
         MFX_CHECK((attrib[1].value & vaRCType), MFX_ERR_DEVICE_FAILED);
 
         attrib[1].value = vaRCType;
+        priority = core.GetSession()->m_priority;
+        if (priority == MFX_PRIORITY_LOW)
+        {
+            attrib[2].value = -512;
+        } else if (priority == MFX_PRIORITY_HIGH)
+        {
+            attrib[2].value = 512;
+        } else
```

Comment:
I'd prefer to have explicit else if (MFX_PRIORITY_NORMAL  == priority) to avoid mysterious behavior in the future if we add new priority policy

Reply:
Agree. Will update it when VA interface change is done.

Reply:
The default value will be used for the normal priority and other priority policy.